lms/djangoapps/instructor/views/api.py

```diff
@@ -1047,6 +1057,7 @@ def re_validate_invoice(obj_invoice):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+@sudo_required
 def get_students_features(request, course_id, csv=False):  # pylint: disable=redefined-outer-name
```

Comment:
I should have clarified - I think that requiring sudo for all these commands probably makes sense. But sudo privileges are revoked on a timer, and once they have been revoked, the UX is confusing - my browser doesn't follow a redirect from these ajax requests.